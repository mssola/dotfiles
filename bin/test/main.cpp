/*
 * Copyright (C) 2015 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (c) 2015 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2015 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2013-2015 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2014-2015 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2015 Miquel Sabaté Solà <mikisabate@gmail.com>
 */

int main(int argc, char *argv[])
{
}

