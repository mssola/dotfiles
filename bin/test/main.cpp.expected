/*
 * Copyright (C) 2015-2022 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (c) 2015-2022 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2015-2022 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2013-2022 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2014-2022 Miquel Sabaté Solà <mikisabate@gmail.com>
 * Copyright (C) 2015-2022 Miquel Sabaté Solà <mikisabate@gmail.com>
 */

int main(int argc, char *argv[])
{
}

